CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,25 @@
+# Contributor Code of Conduct
```

Comment:
@DougManuel this is the same code of conduct from bllflow. Is there any modifications that need to be done to this?

Reply:
Not at this time. Looks good.